src/components/MyNdla/ListResource.tsx

```diff
@@ -137,6 +136,7 @@ const ListResource = ({
   menu,
   variant = "list",
   isLoading = false,
+  listItemTitleElement: TitleElement = "h2",
```

Comment:
er dette nødvendig? skal den noengang være h2?

Reply:
Jeg endret kun på det som var spesifisert i trello-kortet

Reply:
Men hvis det er avklart med ndla at det kan fjernes så er jeg veldig for!